conf/application.conf

```diff
@@ -19,6 +19,7 @@ application.router=prod.Routes
 application.global=uk.gov.hmrc.nisp.config.ApplicationGlobal
 
 application.session.httpOnly=true
+play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9032 www.google-analytics.com data:"
```

Comment:
With this current configuration do the PTA banner and contact-frontend elements still work? They come from different ports locally too.